src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritDynamicUrlProcessor.java

```diff
@@ -180,7 +180,7 @@ private static String regexEscapted(char symbol) {
         char operChar = oper.charAt(0);
         CompareType type = CompareType.findByOperator(operChar);
 
-        logger.trace("==> item:({0}) oper:({1}) text:({2})", new Object[]{item, oper, text});
+        logger.trace("==> item:({}) oper:({}) text:({})", new Object[]{item, oper, text});
```

Comment:
Yea, I see that now. Strange, we should perhaps look into bumping slf4j to a newer version but that might put us in a bit of trouble since core is also bundling a version of slf4j that might conflict.